draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -833,16 +835,14 @@ If any step fails, the SD-JWT is not valid, and processing MUST be aborted. Othe
 
 Note that these processing steps do not yield any guarantees to the Holder about having received a complete set of Disclosures. That is, for some digest values in the Issuer-signed JWT (which are not decoy digests) there may be no corresponding Disclosures, for example, if the message from the Issuer was truncated.
 
-It is up to the Holder how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the End-User when needed.
```

Comment:
I sorta feel like something like this sentence should remain.

Reply:
See https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/481/files#r1817324384 'cause github won't let me make a suggestion on deleted lines.